records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -29,8 +29,10 @@ def __init__(self,
                                     processing_instructions=processing_instructions)
         super().__init__(prep, target_table_details, processing_instructions)
 
+    # TODO: Should this take a loader?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker